rero_ils/dojson/utils.py

```diff
@@ -351,7 +353,11 @@ def get_field_items(value):
 def build_string_from_subfields(value, subfield_selection, separator=' '):
     """Build a string parsing the selected subfields in order."""
     items = get_field_items(value)
-    parts = [value for key, value in items if key in subfield_selection]
+    parts = []
+    # remove special character from string
+    for key, value in items:
```

Comment:
this can be replaced by:
```python
return separator.join([remove_special_characters(value) for key, value in items if key in subfield_selection])
```
It is more pythonic but maybe less easy to read.